### PR TITLE
ncm-metaconfig: httpd: support expires

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Review by ned21 — APPROVED** [review]
Seems fine to me. Is it worth cleaning up the panlint error? And what's the reason the tests are failing?

**Comment by jrha** [comment]
The panlint error is due to an existing use of `format()` within `error()` in another part of the schema.
There is a huge raft of test failures, but the first is:
```
Failed test 'No renderfailure (fail: Failed to render with module httpd/2.4/generic_server: Unable to process template for file metaconfig/httpd/2.4/generic_server.tt (module httpd/2.4/generic_server: file error - metaconfig/httpd/config/expires.tt: not found)'
```

**Comment by jrha** [comment]
Basically, either this PR is missing a TT file, or `expires` shouldn't be added to `to_process`.

File: ncm-metaconfig/src/main/metaconfig/httpd/config/directory_basic.tt [hunk 1]
```diff
@@ -5,7 +5,7 @@ directoryindex [% desc.directoryindex.join(' ') %]
 [%- IF desc.exists('limitrequestbody') %]
 limitrequestbody [% desc.limitrequestbody %]
 [% END -%]
-[%- to_process = ['rewrite', 'handler', 'perl', 'outputfilter', 'env', 'limit', 'proxy', 'wsgi'] -%]
+[%- to_process = ['rewrite', 'handler', 'perl', 'outputfilter', 'env', 'limit', 'proxy', 'wsgi', 'expires'] -%]
 [%- FOREACH p IN to_process -%]
 [%-      IF desc.exists(p) -%]
 [%          INCLUDE "metaconfig/httpd/config/${p}.tt" desc=desc.$p %]
```
Comment (jrha) [inline ncm-metaconfig/src/main/metaconfig/httpd/config/directory_basic.tt:11]: There is no TT file provided to satisfy this.

**Comment by stdweird** [comment]
@jrha thanks for pointing it out. the panlint error is fixed in #1561

**Review by jrha — APPROVED** [review]

**Pull request merged**